### PR TITLE
Check coordinates range

### PR DESCRIPTION
**Pull request by emillon**
This ensures that when we parse a point (x, y), `0 <= x < p` and `0 <= y < p` hold (#3).

### CODE REVIEW

**Comment by emillon** [comment]
This is ready for review @pascutto.

File: p256/parameters.ml [hunk 1]
```diff
@@ -7,3 +7,6 @@ let b =
 let g =
   `Hex
     "046B17D1F2E12C4247F8BCE6E563A440F277037D812DEB33A0F4A13945D898C2964FE342E2FE1A7F9B8EE7EB4A7C0F9E162BCE33576B315ECECBB6406837BF51F5"
+
+let p =
+  `Hex "ffffffff00000001000000000000000000000000ffffffffffffffffffffffff"
```
Comment (pascutto) [inline p256/parameters.ml]: Super minor note here : other parameters are uppercase, we can probably have this one uppercase too.

Comment (emillon) [reply p256/parameters.ml]: let's do this!

**Comment by emillon** [comment]
I'll add a test for y > p as well. If (x, y) is on the curve, (x, y+kp) will be as well, so it's enough to add 2**256 enough times to put `y+kp` in the correct range.

**Comment by emillon** [comment]
Can't solve this for now, I'll merge the check in the meantime.

**Pull request merged**